src/snowflake/snowpark/modin/plugin/_internal/index.py

```diff
@@ -257,7 +270,6 @@ def to_pandas(self) -> native_pd.Index:
         return self._index
 
     @property
-    @is_lazy_check
     def values(self) -> ArrayLike:
         """
         Return an array representing the data in the Index.
```

Comment:
did you mean to not implement values? If you're not implementing it, add the TODO back

Reply:
No, calling to_pandas() is implementing it, since we need to materialize the values anyways